js/src/lib/components/InferenceWidget/shared/helpers.ts

```diff
@@ -1,26 +1,49 @@
 import type { ModelData } from "../../../interfaces/Types";
 import { randomItem, parseJSON } from "../../../utils/ViewUtils";
+import type { WidgetExample } from "./WidgetExample";
 import type { ModelLoadInfo, TableData } from "./types";
 
-export function getSearchParams(keys: string[]): string[] {
+const VALID_QUERY_PARAMS = [
+	// string vals
+	"text",
+	"context",
+	"question",
+	"query",
+	"candidateLabels",
+	// table vals
+	"multiClass",
+	"table",
+	// boolean vals
+	"structuredData",
+];
```

Comment:
Otherwise `VALID_QUERY_PARAM` is a generic `string[]`
```suggestion
] as const;
```

Reply:
superceded by https://github.com/huggingface/hub-docs/pull/1023/commits/c89086b0ebea8a6968c7bc621134982270dbdbe6